sycl/include/sycl/ext/oneapi/experimental/annotated_ptr/annotated_ptr.hpp

```diff
@@ -70,6 +70,10 @@ class annotated_ref {
   static_assert(is_valid_property_list, "Property list is invalid.");
 };
 
+__attribute__((deprecated("T is a non-trivially-copyable type, some operators"
+                          "might invoke the copy constructor!"))) void
+non_trivially_copyable();
```

Comment:
This needs a definition:
```suggestion
non_trivially_copyable() {};
```